Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -702,7 +702,7 @@
 - type: entity
   parent: DrinkGlassBase
   id: DrinkFourteenLokoGlass
-  name: Fourteen loko glass
+  name: fourteen loko glass
```

```suggestion
  name: Fourteen Loko glass
```
This is a joke from a proper noun, and proper nouns should be capitalized.